drivers/wireless/bluetooth/bt_rpmsghci.h

```diff
@@ -0,0 +1,95 @@
+/****************************************************************************
+ * drivers/wireless/bluetooth/bt_rpmsghci.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __DRIVERS_WIRELESS_BLUETOOTH_BT_RPMSGHCI_H
+#define __DRIVERS_WIRELESS_BLUETOOTH_BT_RPMSGHCI_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/compiler.h>
+
+#include <nuttx/wireless/bluetooth/bt_driver.h>
```

Comment:
move the inclusion to include/nuttx/wireless/bluetooth/bt_rpmsghci.h and add the inclusion of nuttx/semphare.h

Reply:
we need this for `enum bt_buf_type_e`

Reply:
You're right, `uint8_t` will be OK here